src/ray/util/util.h

```diff
@@ -302,13 +293,8 @@ inline void unsetEnv(const std::string &name) {
   RAY_CHECK_EQ(ret, 0) << "Failed to unset env var " << name;
 }
 
-inline void SetThreadName(const std::string &thread_name) {
-#if defined(__APPLE__)
-  pthread_setname_np(thread_name.c_str());
-#elif defined(__linux__)
-  pthread_setname_np(pthread_self(), thread_name.substr(0, 15).c_str());
-#endif
-}
+// Set [thread_name] to current thread; if it fails, error will be logged.
```

Comment:
backticks `thread_name`. Specify it works on MacOS and Linux, but not for others (i.e. Windows)

Reply:
I think the prev impl in .cc is intended to be inlined. if you don't do that, maybe we can also move GetThreadName?

Reply:
> Specify it works on MacOS and Linux, but not for others (i.e. Windows)

I should've done that, updated.

Reply:
> I think the prev impl in .cc is intended to be inlined

OK